python/cuml/test/dask/test_kmeans.py

```diff
@@ -42,6 +42,9 @@ def test_end_to_end(nrows, ncols, nclusters, n_parts, client=None):
                                    random_state=10)
 
     wait(X_cudf)
+    wait(X_df)
+
+    print(str(X_df))
```

Comment:
I added this line so that I could troubleshoot the numba-related issue. It will be removed once it's all fixed. First item is getting dask/distributed and dask-cuda aligned once again. Matt Rocklin just submitted a PR to fix this (https://github.com/rapidsai/dask-cuda/pull/129).